locales/fa/base.json

```diff
@@ -150,7 +150,8 @@
     "custom-fields-label-empty-info": "شما افزودن عنوان به یکی از فیلدهای شخصی را فراموش کردید.",
     "are-you-sure-question": "برای این کار اطمینان دارید؟",
     "displaying-entries": "در حال نمایش {{count}} سند",
-    "no-value": "بدون محتوی"
+    "no-value": "بدون محتوی",
+    "quit-unsave-entry": "سوف تفقد دخولك ، هل أنت متأكد؟"
```

Comment:
Please remove the new translations except English. This is up to collaborators to add the correct translations. I see that you have used Google Translate for these which is not correct :)